sites/all/themes/custom/ogp_theme/sass/layouts/_l-home-page.scss

```diff
@@ -1,3 +1,11 @@
 .l-home-page {
+  min-height: 750px;
   margin-top: -90px;
+  background-image: url('../images/cover/cover1.jpg');
```

Comment:
You cant put the background image here. Overides all backgrounds and appears above slides. You need to append to each slide.